chronosphere/notification_policy_data.go

```diff
@@ -178,9 +185,21 @@ func notifierListMapToModel(notifierList RoutesNotifierList) (*configmodels.Rout
 	return &configmodels.RoutesNotifierList{
 		NotifierSlugs:      sliceutil.Map(notifierList.Notifiers, func(n *RoutesNotifierListNotifier) string { return n.Slug }),
 		RepeatIntervalSecs: durationInSecs,
+		GroupBy:            notifierListToGroupByModel(notifierList),
```

Comment:
nit: update name + make this method work purely on the GroupBy field, e.g., we should be passing `notifierList.GroupBy` here, not the whole object since it only needs the GroupBy field. The `toModel` and `fromModel` should be symmetric.

name should probably be `notifierListGroupByToModel` to align with the naming used in the rest of the code.

Reply:
thanks, updated